js/precision_store_points.js

```diff
@@ -0,0 +1,37 @@
+var store_points_var = false;
```

Comment:
This file is not the most recent precision_store_points from my latest commit to the www branch. What version of www are you using? I seperated this code into a new file precision.js required for webgazer's build system.